README.md

```diff
@@ -120,7 +120,7 @@ ActiveArea(
 
 And that's it!
 
-### Specialised Actors
+### Specialized Actors
```

Comment:
This is not a typo. This is how we spell it in South Africa :) But will merge it in anyway haha